app/src/main/res/layout-sw600dp/topic_practice_fragment.xml

```diff
@@ -23,4 +23,4 @@
     app:layout_constraintEnd_toEndOf="parent"
     app:layout_constraintStart_toStartOf="parent"
     app:layout_constraintTop_toTopOf="parent" />
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done.